public/static/covizu/index.html

```diff
@@ -210,18 +210,17 @@
             <h1><span onclick="$('#splash').dialog('open');"><small><i>open</i></small>CoVizu</span></h1>
           </div>
           <div id="covizu-i18n" style="font-size: 8pt;">
-            <a href="index.html">en</a>
-            <a href="index-es.html">es</a>
-            <a href="index-fr.html">fr</a>
-            <a href="index-zh.html">zh</a>
+            <a href="index.html" id="covizu-18n__en">en</a>
```

Comment:
are these needed to be an ID? you could also (better?) do data tags, and use css attribute selectors